includes/gateways/stripe/includes/payment-methods/class-give-stripe-checkout.php

```diff
@@ -363,9 +366,15 @@ public function redirect_to_checkout() {
 				processingHtml.innerHTML = '<div class="give-stripe-checkout-processing-container" style="position: absolute;top: 50%;left: 50%;width: 300px; margin-left: -150px; text-align:center;"><div style="display:inline-block;"><span class="give-loading-animation" style="color: #333;height:26px;width:26px;font-size:26px; margin:0; "></span><span style="color:#000; font-size: 26px; margin:0 0 0 10px;">' + give_stripe_vars.checkout_processing_text + '</span></div></div>';
 
 				window.addEventListener('load', function() {
-					const stripe = Stripe( '<?php echo $publishable_key; ?>', {
+					let stripe = {};
+
+					stripe = Stripe( '<?php echo $publishable_key; ?>' );
```

Comment:
Have you tested the per-form account functionality with multiple donation forms on a page using different Stripe accounts? As well, testing the form grid functionality to verify it works as expected? Those are going to be very popular use cases.

Reply:
@DevinWalker I have noticed a couple of minor issues which I fixed with the latest commits. Thanks for pointing out these scenarios.